website/src/types/Conversation.ts

```diff
@@ -1,4 +1,16 @@
-export interface Message {
+export type EmojiOp = "add" | "remove" | "toggle";
+
+export interface MessageEmoji {
+  name: string;
+  count: number;
+}
+
+export interface MessageEmojis {
+  emojis: { [emoji: string]: number };
+  user_emojis: string[];
+}
+
+export interface Message extends MessageEmojis {
```

Comment:
the `Message` interface is what gets received from the backend, does the backend also send the emojis now?

if not, please create a separate types that combines message & emojis.

Reply:
Yep, we get emojis for all messages.